config/packages/security.yaml

```diff
@@ -32,4 +32,5 @@ security:
     access_control:
         # this is a catch-all for the admin area
         # additional security lives in the controllers
-        - { path: '^/(%app_locales%)/admin', roles: ROLE_ADMIN }
+        - { path: '^/bolt', roles: ROLE_ADMIN }
```

Comment:
We shouldn't hardcode the path to `/bolt. Right now we have it defined in `routes.yaml` and the annotations like `@Security("has_role('ROLE_ADMIN')")` in `src/Controller/Backend` make sure of the access control. If we do that here, it will make it easier for users to mess up by changing one, and leacing this as-is, leaving their install wide open to the world.

Reply:
so can we move it to a variable?

Reply:
Yes, if you think that's better than keeping it in routing, i have no problems with it.

Reply:
See #245